src/device-registry/utils/distance.js

```diff
@@ -1,13 +1,47 @@
 const { logObject, logElement, logText } = require("./log");
-const distanceBtnTwoPoints = (latitude1, longitude1, latitude2, longitude2) => {
-  try {
+
+const distance = {
```

Comment:
refactored distance utill with new functionality from deleted utils that were focused on distance calculations.